### PR TITLE
FEATURE: Eliminate purchase notification popups

### LINKED ISSUE #2: FEATURE: Eliminate purchase notification popups
# User Story

* As a User (especially mobile)
* I can sell cows without seeing toast message pop up.
* So that I can sell cows more easily

### Discussion

Right now when selling many cows by spam-clicking, the screen floods with times "Cow sold!" popups:

image

This is especially distracting on mobile, as the popups block the interface and buttons.

These toast message aren't adding any value for the user, so we should just eliminate them in the case where the
cow sale was successful.  The messages may still be useful if/when there is an error condition, but if the purchase
is successful, it will be reflected in the cow count and dollar amounts on the screen.

(The toast message is a leftover feature from before those numbers were being reliably displayed and updated... it was useful at the time, but isn't anymore.)

### Acceptance Criteria

- [x] No Toast message is generated when selling cows.

### PR DESCRIPTION
**Pull request by kaitntom**
## Overview

In this PR, I removed the "Cow sold!" toast notification when a cow is sold.

## Screenshots (Optional)

Before
Screenshot 2023-11-20 at 5 10 30 PM
After
Screenshot 2023-11-21 at 1 20 56 PM

## Validation (Optional)

1. Download this branch.
2. Run the project.
3. Go to this page.
4. Hit sell cow
5. Watch as the notification "Cow sold!" does not pop-up.

## Tests

- [x] Frontend Unit tests (`npm test`) pass
- [x] Frontend Test coverage (`npm run coverage`) 100%
- [x] Frontend Mutation tests (`npx stryker run`) 100% 

## Linked Issues

Closes #2

### CODE REVIEW

**Comment by bobert-2** [comment]
LGTM!

**Review by bobert-2 — DISMISSED** [review]
Looks good :)

**Comment by madisonlong1** [comment]
LGTM

**Review by madisonlong1 — DISMISSED** [review]
LGTM

File: frontend/src/main/pages/PlayPage.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, {useState} from "react";
 import { Container, CardGroup, Button } from "react-bootstrap";
 import { useParams } from "react-router-dom";
-import { toast } from "react-toastify";
+//import { toast } from "react-toastify";
```
Comment (pconrad) [inline frontend/src/main/pages/PlayPage.js]: Please see: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/pages/PlayPage.test.js [hunk 2]
```diff
@@ -94,7 +94,7 @@ describe("PlayPage tests", () => {
 
         await waitFor(() => expect(axiosMock.history.put.length).toBe(2));
 
-        expect(mockToast).toBeCalledWith("Cow sold!");
+        //expect(mockToast).toBeCalledWith("Cow sold!");
```
Comment (pconrad) [inline frontend/src/tests/pages/PlayPage.test.js]: Please see: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

**Review by pconrad — CHANGES_REQUESTED** [review]
Please see: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/main/pages/PlayPage.js (hunk 1 above)
Comment (pconrad) [inline frontend/src/main/pages/PlayPage.js]: Please see: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/main/pages/PlayPage.js [hunk 3]
```diff
@@ -90,7 +90,7 @@ export default function PlayPage() {
 
 
   const onSuccessSell = () => {
-    toast(`Cow sold!`);
+    //toast(`Cow sold!`);
```
Comment (pconrad) [inline frontend/src/main/pages/PlayPage.js]: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/pages/PlayPage.test.js (hunk 2 above)
Comment (pconrad) [inline frontend/src/tests/pages/PlayPage.test.js]: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**